curriculum/challenges/english/blocks/es-a1-review-greetings-and-farewells/68c850ed23d494c9be5cb22a.md

```diff
@@ -1,58 +1,89 @@
 ---
 id: 68c850ed23d494c9be5cb22a
-title: "Dialogue 1: I'm Tom"
-challengeType: 21
-dashedName: dialogue-1-im-tom
+title: Grammar Highlights
+challengeType: 31
+dashedName: grammar-highlights
 ---
 
+<!-- GRAMMAR -->
+
 # --description--
 
-Watch the video below to understand the context of the upcoming lessons.
+`¡Felicitaciones!` Congratulations on completing your first **Learn** and **Practice** sections!
+
+You've now reached the **Review** section of this module. Every module has a review section. So, this is a great place to revisit whenever you want to refresh your memory about the topics you just saw. 
+
+The task below is called **Grammar Highlights**, and its goal is to summarize the key grammar you've covered in this module.
+
+After reading it, just mark the task as complete and continue to the next part of the review.
+
+## Grammar Highlights
+
+### Subject Pronoun Omission
+
+In Spanish, it's common to omit the subject pronoun (like `yo`, `tú`, etc.) because the verb form already tells who the subject is. For example:
+
+- `Soy Camila.` – I am Camila.  
+
+- `Me llamo Esteban.` – My name is Esteban.
+
+<br />
+
+### Using `soy` to Talk About Identity
+
+The verb `ser` (to be) is used to describe who you are; Like your name or profession.
+
+`Soy` means "I am". It's often followed directly by a noun and no article is needed. For example:
+
+- `Soy diseñadora.` – I am a designer.  
+
+- `Soy analista.` – I am an analyst.
+
+### No Articles Before Professions
+
+In English, people say "a developer", but in Spanish, you **don't use** an article (`un` or `una`) before a profession.
```

Comment:
This certification will be translated into different languages, right? Considering that some learners may not have learned English, we'd better avoid using "in English".